uns_plugin/schema_validation/validator_test.go

```
@@ -0,0 +1,633 @@
+         },
+         "additionalProperties": false
+      }
+   },
+   "required": ["virtual_path", "fields"],
+   "additionalProperties": false
+}`)
+		var validator *Validator
+
+		BeforeEach(func() {
+			validator = NewValidator()
+			validator.LoadSchema("_sensor_data", 1, schema)
```

golangci-lint (1.64.8)

Error return value of `validator.LoadSchema` is not checked

(errcheck)